### DISCUSSION TITLE
File not found error when the file exists (Pacman)

### DISCUSSION THREAD

**Discussion by nihaals** [community/community#26089] [Actions]
<p>The file is found with a glob and even <code>$ ls</code>, yet apparently doesn’t exist? I can’t reproduce this in my own container and am fairly confident it’s not an issue with <code>pacman</code> (I see no similar reported issues, can’t reproduce in a local Docker container).</p>
<p><img src="https://user-images.githubusercontent.com/18350092/181095169-e561acc4-14cf-4fd3-98a2-98804d0ef480.png" alt="image" data-base62-sha1="zZrinuakPYMRpwldVT4NKTcQgXo" width="637" height="211"></p>
<ul>
<li><a href="https://github.com/nihaals/visual-studio-code-insiders-arch/runs/986142382?check_suite_focus=true#step:6:1" rel="nofollow noopener">Relative path</a></li>
<li><a href="https://github.com/nihaals/visual-studio-code-insiders-arch/runs/986180749?check_suite_focus=true#step:6:1" rel="nofollow noopener"><code>cd</code>ing and <code>ls</code>ing</a></li>
<li>Absolute path: <code>/nihaals/visual-studio-code-insiders-arch/runs/986222931?check_suite_focus=true#step:6:1</code>
</li>
</ul>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/nihaals">@nihaals</a>,</p>
<p>Glad to see you in Github Community Forum!</p>
<p>The issue should be due to the artifacts are not download properly. The download package size is only 4096 which is not correct.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d29052e0871c1688a416f995c7a53eaa3456e61e.png" data-download-href="/uploads/short-url/u2JtOSwJchmvhoObTs0jo4CM44S.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181095170-c4b3c15f-e83c-41c9-b73f-411ac0667d7e.png" alt="image" data-base62-sha1="u2JtOSwJchmvhoObTs0jo4CM44S" width="690" height="191" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d29052e0871c1688a416f995c7a53eaa3456e61e_2_690x191.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d29052e0871c1688a416f995c7a53eaa3456e61e_2_1035x286.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/d/d29052e0871c1688a416f995c7a53eaa3456e61e_2_1380x382.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d29052e0871c1688a416f995c7a53eaa3456e61e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1737×481 32.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I notice you have download the artifacts to another folder ‘pkg’, and then copy the package to <code>dist</code> for command <code>repo-add</code>, it resolved the issue, confirmed on my side.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/7/70e4c06d971430f12e78d382f68b39cc09d84b67.png" data-download-href="/uploads/short-url/g6HzDRPQcWPfZkgqKngwpYLKCiP.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181095171-8aff5c3e-30fe-4bd5-ae3c-b12adcd5c7cd.png" alt="image" data-base62-sha1="g6HzDRPQcWPfZkgqKngwpYLKCiP" width="690" height="206" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/7/70e4c06d971430f12e78d382f68b39cc09d84b67_2_690x206.png, https://aws1.discourse-cdn.com/github/optimized/2X/7/70e4c06d971430f12e78d382f68b39cc09d84b67_2_1035x309.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/7/70e4c06d971430f12e78d382f68b39cc09d84b67_2_1380x412.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/7/70e4c06d971430f12e78d382f68b39cc09d84b67_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1735×520 35.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Besides, you can raise an issue <a href="https://github.com/actions/download-artifact/issues" rel="nofollow noopener">here</a> for action ‘actions/download-artifact’, dev team will help to check and fix.</p>
<p>Thanks</p>

**Comment by nihaals** [comment, marked as answer]
<p>I somehow managed to miss the <em><strong>bold</strong></em> text in the docs saying that when downloading all artefacts they are placed in subdirectories and I was attempting to use the directory as a file</p>

**Comment by weide-zhou** [comment]
<p>Thanks <a class="mention" href="/nihaals">@nihaals</a>! :grinning:</p>
<p>Checked the file details:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/dd10e7fb9d993a1f55c1bd833075efdf59b654de.png" data-download-href="/uploads/short-url/vxDK1oKxsPMTORSo0cPUdHlZl9k.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181095175-1801752f-f7b8-4425-831f-d7c7af0d9c7d.png" alt="image" data-base62-sha1="vxDK1oKxsPMTORSo0cPUdHlZl9k" width="690" height="93" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/dd10e7fb9d993a1f55c1bd833075efdf59b654de_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1065×144 5.87 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>And the <strong>bold</strong>   text <a href="https://github.com/actions/download-artifact#download-all-artifacts" rel="nofollow noopener">here</a> for any guy who encounter same problem:<br>
If the  <code>name</code>  input parameter is not provided, all artifacts will be downloaded.  <strong>To differentiate between downloaded artifacts, a directory denoted by the artifacts name will be created for each individual artifact.</strong></p>
